### ISSUE TITLE
Adding timespan causes integer overflow

### ISSUE THREAD

**Issue by eriktjacobsen** [mbossenbroek/simple-time#5]
Think this should be bigint:

``` clojure
project.core=> (stime/+ (stime/utc-now) (stime/days->timespan 59))

     clojure.core/eval                          core.clj: 2927
                   ...                                        
pipeline.core/eval4138  form-init8192743386029455111.clj:    1
                   ...                                        
    simple-time.core/+                          core.clj:  425
                   ...                                        
java.lang.IllegalArgumentException: Value out of range for int: 5097600000
```

I do realize `(stime/add-days (stime/utc-now) 59)` works as well, but figure this bug should still be fixed.

**Comment by mbossenbroek** [comment]
That actually seems to be a limitation of joda time: http://www.joda.org/joda-time/apidocs/org/joda/time/LocalDateTime.html#plusMillis(int)

The underlying method only takes an int. Off the top of my head, I could either add Integer/MAX_VALUE over & over until adding the total number of requested milliseconds. Or I could convert it into ms/seconds/minutes/hours & add those instead.

Any preference (or other ideas)?

**Comment by mbossenbroek** [comment]
Sorry this took so long. Fixed in version 0.2.1
